### ISSUE TITLE
Right-click on wide error message doesn't spawn context menu under mouse

### ISSUE THREAD

**Issue by AdriaandeJongh** [godotengine/godot#86198]
### Tested versions

Reproducible in Godot v4.2.1.stable.mono, haven't checked other versions

### System information

Godot v4.2.1.stable.mono - macOS 14.2.0 - Vulkan (Mobile) - integrated Apple M3 Max - Apple M3 Max (14 Threads)

### Issue description

Video of this behavior:
https://github.com/godotengine/godot/assets/5611323/c6a9b108-1879-422a-9687-2e716802559f

The context menu spawns all the way to the left instead of under the mouse. Expected behavior is under the mouse.

### Steps to reproduce

Find yourself a long error and right click it!

### Minimal reproduction project (MRP)

N/A

**Comment by AThousandShips** [comment]
Can you confirm this with `interface/editor/single_window_mode` under editor settings enabled?

**Comment by AdriaandeJongh** [comment]
no, single window mode is disabled. regardless of that toggle, I would expect the right-click context menu to always show up under the mouse.

**Comment by AThousandShips** [comment]
That's not what I asked, I asked you to try it to help investigating the bug 🙂

**Comment by AdriaandeJongh** [comment]
After enabling Single Window Mode, the context menu does show up under the mouse.

**Comment by AThousandShips** [comment]
Good, that indicates that the issue is somehow in translating the position of the popup as a dedicated window, as it works with embedded windows
